corehq/apps/hqwebapp/views.py

```diff
@@ -83,6 +86,11 @@ def couch_check():
         return isinstance(results, list), None
 
 
+def is_deploy_in_progress():
+    cache = get_redis_default_cache()
```

Comment:
this just does the getting so we don't have to worry about the expires/setting here. i was leveraging the existing `CACHES` variable. direct connection makes sense to me, can do. we should probably surround in try catch as well and return false on error